domains/deadlyranger.json

```
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "DeadlyRanger",
+        "email": "padaikrle13@gmail.com"
+      },
+
+      "record": {
+        "URL": "10"
```

Perhaps you wanted to redirect to your website?
```suggestion
        "URL": "https://deadlyranger.gq"
```